build/MSBuildExtensions.targets

```diff
@@ -63,6 +63,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <WriteLinesToFile File="$(GeneratedMSBuildExtensionsDirectory)/$(BundledVersionsPropsFolder)/$(BundledVersionsPropsFileName)"
                       Lines="$(BundledVersionsPropsContent)"
                       Overwrite="true" />
-
+    <Copy
```

Comment:
I want to copy for now. See my description of the PR above. We can stop generating the other one later, once we have the resolver change in.